assets/js/membraneWebRTC.ts

```diff
@@ -362,20 +412,109 @@ export class MembraneWebRTC {
    *   })
    *   .then((stream) => {
    *     let videoTrack = stream.getVideoTracks()[0];
-   *     webrtc.replaceTrack(oldTrackId, videoTrack);
+   *     webrtc.replaceTrack(oldTrack, videoTrack);
    *   })
    *   .catch((error) => {
    *     console.error('Error switching camera', error);
    *   })
    * ```
    */
-  public async replaceTrack(oldTrackId: string, newTrack: MediaStreamTrack): Promise<any> {
-    const sender = this.connection!.getSenders().find((sender) => {
-      return sender!.track!.id === oldTrackId;
+  public async replaceTrack(
+    oldTrackId: string,
```

Comment:
I would call this just a `trackId` as we are not generating a new track id. Also docs for this function need to be fixed. Please check docs for other functions too